<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon/res.qrc">
    <normaloff>:/case/open_folder.png</normaloff>:/case/open_folder.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="edt_search">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_expandAll">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icon/res.qrc">
           <normaloff>:/case/expand.png</normaloff>:/case/expand.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_collapseAll">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icon/res.qrc">
           <normaloff>:/case/collapse.png</normaloff>:/case/collapse.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="tw_category">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <column>
        <property name="text">
         <string>Category</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Category Id</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_addCategory">
     <property name="text">
      <string>Category 추가</string>
     </property>
     <property name="icon">
      <iconset resource="icon/res.qrc">
       <normaloff>:/case/add_category.png</normaloff>:/case/add_category.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_addCategory">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon/res.qrc">
     <normaloff>:/case/add_category.png</normaloff>:/case/add_category.png</iconset>
   </property>
   <property name="text">
    <string>Category 추가</string>
   </property>
   <property name="toolTip">
    <string>Category 추가</string>
   </property>
   <property name="statusTip">
    <string>New Test Case</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
